demo/src/index.tsx

```diff
@@ -16,19 +18,33 @@ ReactDOM.render(
       onPageView={params => {
         const path = window.location.pathname + window.location.search;
         console.info('✅GA: Default PageView', path, params); // NOTE: Google Analytics(GA4)는 기본적으로 페이지뷰가 적용됩니다 - 따로 추가 필요X
+        // NOTE: Google Analytics(GA4)는 기본적으로 페이지뷰가 적용됩니다 - 따로 추가 필요X
         fruitLogger.pageView(path, params);
+        toaster.showPageViewToast(path, params);
```

Comment:
```suggestion
        fruitLogger.pageView(path, params);
        toaster.pageView(path, params);
```

fruitLogger와 메소드명을 통일해도 좋을 것 같아요!

Reply:
@greatSumini
- 우선 모바일은 확인 안했습니다. 당장 모바일에서 필요해보이지는 않아서요. 혹여나 문제가 생긴다면 그때 또 트러블슈팅해볼게요~ (생각지 못했는데 감사합니다.) 
- 말씀해주신대로 메소드는 기존 logger랑 맞추겠습니다! 그게 좋을 것 같습니다.